staging/src/k8s.io/apiserver/pkg/storage/value/transformer_test.go

```diff
@@ -99,3 +103,135 @@ func TestPrefixTo(t *testing.T) {
 		}
 	}
 }
+
+func TestPrefixFromMetrics(t *testing.T) {
+	testErr := fmt.Errorf("test error")
+	transformerErr := fmt.Errorf("test error")
+	identityTransformer := PrefixTransformer{Prefix: []byte{}, Transformer: &testTransformer{from: []byte("value1")}}
```

Comment:
Why not use the actual identity transformer.
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/identity/identity.go

Reply:
the context we used in test is not implementing https://github.com/kubernetes/kubernetes/blob/70ed851360e89742a6ed020b4d71499e1913af46/staging/src/k8s.io/apiserver/pkg/storage/value/transformer.go#L32.